### PR TITLE
Include .gitignore in starters

### PR DESCRIPTION
**Pull request by MGatner**
**Description**
devstarter and appstarter both have very old versions of .gitignore - it isn't getting updated when new deployments are made. This PR adds .gitignore to the list of files copied out during a release.

**Checklist:**
- [X] Securely signed commits

### CODE REVIEW

File: admin/release-appstarter [hunk 1]
```diff
@@ -12,7 +12,7 @@ git checkout $branch
 echo -e "${BOLD}Build the framework distributable${NORMAL}"
 
 echo -e "${BOLD}Copy the main files/folders...${NORMAL}"
-releasable='app public writable README.md contributing.md env license.txt spark tests/_support'
+releasable='app public writable README.md contributing.md env license.txt spark tests/_support .gitignore'
```
Comment (jim-parry) [inline admin/release-appstarter]: tests_support should actually not be in this list (my mistake). If you could remove it, that would save a change later :)

**Comment by jim-parry** [comment]
Good idea. One teeny change requested :)

File: admin/release-appstarter (hunk 1 above)
Comment (MGatner) [reply admin/release-appstarter]: Sure thing, all set!

**Comment by jim-parry** [comment]
I have merged this, but I don't think that the gitignore in the framework release prep will make any difference. It will end up in vendor/codeigniter4/framework, which will have no effect on a developer's project.

**Comment by MGatner** [comment]
How does https://github.com/codeigniter4/devstarter get seeded? That's what really needs the updated files.

**Comment by jim-parry** [comment]
It doesn't ... any updates are done manually when there are app changes in a release.

**Comment by MGatner** [comment]
Ah thanks for clarifying. I misunderstood that deployment script - sorry for the trouble!

**Comment by jim-parry** [comment]
The appstarter and framework repos are updated as a "normal" part of a release,
That's where the admin/release-appstarter and release-framework scripts are invoked.
This is all done on my system, and then the releases are manually tagged on github.
Not all can be automated, but as much as possible has been :)

**Comment by jim-parry** [comment]
See admin/README.md and admin/workflow.md

**Pull request merged**